### DISCUSSION TITLE
Bulk export with s3

### DISCUSSION THREAD

**Discussion by melvinmihaylovnulabg** [pxlrbt/filament-excel#205] [Q&A]
<img width="552" alt="image" src="https://github.com/user-attachments/assets/d7742ece-5ddc-4a90-9b3f-32b4c0edcb11">
Is it possible to bulk export using the s3 instead of temp storage?

**Comment by pxlrbt** [comment]
Sorry, for digging out this old discussion.

I think this is possible by overwriting the disk used by the plugin:
`config()->set('filesystems.disks.filament-excel', [...])`
